src/main/scala/com/typesafe/sbt/jse/SbtJsEngine.scala

```diff
@@ -87,24 +91,27 @@ object SbtJsEngine extends AutoPlugin {
       val npmDirectory = baseDirectory.value / NodeModules
       val npmPackageJson = baseDirectory.value / PackageJson
       val cacheDirectory = streams.value.cacheDirectory / "npm"
+      implicit val timeout = Timeout(npmTimeout.value)
+      val webJarsNodeModulesPath = (webJarsNodeModulesDirectory in Plugin).value.getCanonicalPath
+      val nodePathEnv = LocalEngine.nodePathEnv(immutable.Seq(webJarsNodeModulesPath))
+      val engineProps = engineTypeToProps(engineType.value, command.value, nodePathEnv)
+      val nodeModulesDirectory = (webJarsNodeModulesDirectory in Plugin).value
+      val logger = streams.value.log
+      val currentState = state.value
```

Comment:
Most of these (and the changes below) are just moving the use of the `value` macro out of a function body because sbt 1.0 doesn't allow that, whereas sbt 0.13 did. It has no actual impact on what happens, because in sbt 0.13 a semantically similar transformation was done by the macro anyway.